GitUI/UserControls/FilterToolBar.cs

```diff
@@ -305,6 +282,31 @@ private void UpdateBranchFilterItems()
                 await this.SwitchToMainThreadAsync();
                 BindBranches(branches);
             }).FileAndForget();
+
+            return;
+
+            void BindBranches(string[] branches)
+            {
+                var autoCompleteList = tscboBranchFilter.AutoCompleteCustomSource.Cast<string>();
+                if (!autoCompleteList.SequenceEqual(branches))
+                {
+                    tscboBranchFilter.AutoCompleteCustomSource.Clear();
+                    tscboBranchFilter.AutoCompleteCustomSource.AddRange(branches);
+                }
+
+                string filter = tscboBranchFilter.Items.Count > 0 ? tscboBranchFilter.Text : string.Empty;
```

Comment:
```suggestion
                var filter = tscboBranchFilter.Items.Any() ? tscboBranchFilter.Text : string.Empty;
```

Reply:
The coding guidelines (that are not written down...) recommends explicit types.